tests/serializers/test_serializers.py

```diff
@@ -56,6 +63,20 @@ def test_multi_level(self):
         self.assertIn('/parent2/1/child2/1/grandchild1/2/', data['second'][0]['grand'][1]['url'])
         self.assertIn('/parent2/1/child2/1/grandchild1/3/', data['second'][0]['grand'][2]['url'])
 
+    def test_multi_level_parent_ref(self):
+        url = reverse('grandchild1-detail', kwargs={'root_pk': 1, 'parent_pk': 2, 'pk': 1})
+        data = self.get_json_response(url)
+        self.assertIn('/parent2/1/child2/1/', data['parent'])
+
+    def test_multi_level_parent_ref_reverse(self):
+        url = reverse('grandchild1-list', kwargs={'root_pk': 1, 'parent_pk': 2})
+        data = self.post_json_request(url, {
+            'name': 'NewChild',
+            'parent': reverse('child2-detail', kwargs={'root_pk': 1, 'pk': 2})
+        })
+        self.assertEqual(data['name'], 'NewChild')
+
+
```

Comment:
Travis barked on flake8 for this line, and I am lazy enough to ask you to remove it for me. Please?